auxiliary/RewardsDistributor/cannonfile.toml

```diff
@@ -38,7 +33,6 @@ artifact = "RewardsDistributor"
 args = [
     "<%= settings.rewardManager %>",
     "<%= settings.poolId %>",
-    "<%= settings.collateralType %>",
```

Comment:
Can we keep args without breaking change and accept zero address instead please

Reply:
its not a breaking change because its for the constructor of the contract, and the functionality of the contract has significantly changed in an of itself. It might as well be a new contract. since the contract no longer pays any attention to collateral type, I went ahead and removed the arg. On the contrary, I think this should be useful in creating awareness of the change downstream in case its forgotten about.

Reply:
Got it, thanks